app-catalog/src/components/charts/List.tsx

```diff
@@ -16,15 +17,22 @@ import {
   TextField,
   Tooltip,
   Typography,
+  useTheme,
 } from '@mui/material';
 import { Autocomplete, Pagination } from '@mui/material';
 import { useEffect, useState } from 'react';
 //import { jsonToYAML, yamlToJSON } from '../../helpers';
 import { fetchChartsFromArtifact } from '../../api/charts';
-import CNCFLight from './cncf-icon-color.svg';
 //import { createRelease } from '../../api/releases';
 import { EditorDialog } from './EditorDialog';
 
+interface AppCatalogConfig {
+  showVerified: boolean;
+}
+
+export const store = new ConfigStore<AppCatalogConfig>('app-catalog');
+const appCatConf = store.useConfig();
```

Comment:
appCatConf is a hook, hooks names have to start with `use`. something like `useAppCatalogConfig`

Reply:
done!

Reply:
This now has a name of `storeConfig` but it is a hook, so should have the "use" in front. Probably `useStoreConfig` is a good name.